src/shared/components/annotation/Civic.spec.tsx

```diff
@@ -22,16 +21,10 @@ describe('Civic with no data', () => {
     });
 
     it('displays a load spinner when there is no civic data', () => {
-        const spinner = component.find(Circle);
+        const spinner = component.find(".fa-spinner");
```

Comment:
`react-mutation-mapper` is now using a simple `fa-spinner` instead of a standalone `Circle` component